pythonturtle/turtleprocess.py

```diff
@@ -51,11 +53,44 @@ def send_report(self):
 
     def run(self):
 
-        builtins.help = builtins.license = builtins.exit = \
-                                lambda *args, **kwargs: print('Not supported')
-
         self.turtle = Turtle()
 
+        def exit():
+            """
+            The built-in exit function terminates the interpreter hence
+            making the application unusable. We terminate the
+            application directly instead. NOTE: We use a proxy function
+            to hide the implementation details from the user.
+            """
+            evt = wx.CommandEvent(wx.ID_EXIT, wx.ID_EXIT)
+            wx.PostEvent(self.window, evt)
```

Comment:
I want to fire an "exit the application" event. Unfortunately, this doesn't work. Any idea, @cool-RR?

Unfortunately, brute-force attempts like `wx.Abort()` behave just like `os._exit(0)`, which makes the application window hang. :worried:

Reply:
How about calling the `.Close` method on the `Frame`?

Reply:
It does nothing. It even returns `True`, which translates to "[event was handled](https://docs.wxpython.org/wx.Window.html#wx.Window.Close)". I also tried with `force=True`, it behaves the same:
```
self.window.Close(force=true)
```

Reply:
I could investigate but it's not worth the time. As far as I'm concerned you can make `exit` just do nothing. Honestly this is very esoteric. If you're doing this for fun you can just ask on the wxPython forum and you're likely to get an answer.